src/omnibar/omnibar.ts

```diff
@@ -71,6 +75,10 @@ let omnibarConfig: BBOmnibarConfig;
 let currentLegacyKeepAliveUrl: string;
 let promiseResolve: () => void;
 let pushNotificationsConnected: boolean;
+let unreadNotificationCount: number;
+let serviceName: string;
+let currentTitleParts: string[] = [];
+let titleSet: boolean = false;
```

Comment:
Do you need a separate `titleSet` variable, or can we just check to see if `currentTitleParts` is not `undefined`?